src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/resource/PlansController.kt

```diff
@@ -26,22 +26,57 @@ import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.con
 import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.model.IdentifiedNeed
 import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.model.Plan
 import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.model.request.CreateIdentifiedNeedRequest
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.model.request.CreatePlanRequest
 import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.model.request.UpdateIdentifiedNeedRequest
 import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.model.request.UpsertPlanRequest
 import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.service.PlanService
 import uk.gov.justice.hmpps.kotlin.common.ErrorResponse
 import java.util.UUID
 
 @RestController
-@RequestMapping(
-  path = ["/csip-records"],
-  produces = [MediaType.APPLICATION_JSON_VALUE],
-)
-@Tag(
-  name = "6. Plans Controller",
-  description = "Endpoints for Plans And Identified Needs operations",
-)
+@RequestMapping(path = ["/csip-records"], produces = [MediaType.APPLICATION_JSON_VALUE])
+@Tag(name = "6. Plans Controller", description = "Endpoints for Plans And Identified Needs operations")
 class PlansController(private val planService: PlanService) {
+  @ResponseStatus(HttpStatus.CREATED)
+  @PostMapping("/{recordUuid}/plan")
+  @Operation(
+    summary = "Create the CSIP plan",
+    description = "Create the CSIP plan. Publishes person.csip.record.updated event with affected component of Plan and IdentifiedNeed",
```

Comment:
Would you mind fixing the "IdentifiedNeed" -> "Identified Need(s)" in the next PR?